e2e/_suites/fleet/fleet.go

```diff
@@ -1315,12 +1329,15 @@ func (fts *FleetTestSuite) theMetricsInTheDataStream(name string, set string) er
 					"dataset": "system." + set,
 					"enabled": "true",
 					"type":    name,
-				}).Info("The " + name + "with value system." + set + " in the metrics")
+					"os":      os,
+				}).Info("The " + name + " with value system." + set + " in the metrics")
 
 				if int64(int64(item.Path("last_activity_ms").Data().(float64))) > startTime {
-					log.WithField(
-						"Activity Time stamp for the "+name+"system."+name, "Is valid",
-					).Info("The " + name + "with value system." + set + " in the metrics")
+					log.WithFields(log.Fields{
+						"last_activity_ms": item.Path("last_activity_ms").Data().(float64),
+						"startTime":        startTime,
+						"os":               os,
+					}).Info("The " + name + " with value system." + set + " in the metrics")
```

Comment:
For upcoming PRs, the preferred logging style should be something like this:
```golang
log.WithFields(log.Fields{
	"last_activity_ms": item.Path("last_activity_ms").Data().(float64),
	"startTime":        startTime,
	"os":        os,
	"integration":   name,
	"metricset":       set,
}).Info("Last activity greater than start time")
```
And maybe using Debug level.